product_harmonized_system/views/product_product.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="product_variant_easy_edit_view" model="ir.ui.view">
+        <field name="name">product.product.view.form.easy</field>
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_variant_easy_edit_view" />
+        <field name="arch" type="xml">
+            <sheet position="inside">
+                <group
+                    name="info"
+                    string="General Information"
```

Comment:
High chance to have another group named `info` (quite used) by another module, is there no other existing page/group to use to add this field? Or maybe rename this group?

Reply:
I propose to change this to name="intrastat-extrastat"